cmd/tester/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM golang:1.9.2 as builder
+RUN mkdir -p /go/src/github.com/openfaas-incubator/connector
+WORKDIR /go/src/github.com/openfaas-incubator/connector
```

Comment:
The path I guess is `connector-sdk` right? To match the repo.

Reply:
Actually I think it's:

`/go/src/github.com/openfaas-incubator/connector-sdk/cmd/tester/` correct?